doc/Command-Reference.md

```diff
@@ -5121,6 +5121,50 @@ This command is to display the link-training status of the selected interfaces.
     Ethernet8      trained          on      up       up
   ```
 
+**show interfaces mac-statistics**
+
+The show interface mac-stats command provides detailed statistics and error counters for MAC-level operations on an interface. It displays the status of various operational parameters, error counts, and timestamps for when these errors occurred.
+
+- Usage:
+  ```
+  show interfaces mac_statistics [<interface_name>]
+  ```
+
+- Example:
+  ```
+  admin@sonic:~$ show interfaces mac_statistics Ethernet4
+  Field                             Value
+  --------------------------------  -------------------
+  oper_error_status                 5442
+  mac_local_fault_count             2
+  mac_local_fault_time              2024-11-02 04:00:05
+  fec_sync_loss_count               2
+  fec_sync_loss_time                2024-11-02 04:00:05
+  fec_alignment_loss_count          2
+  fec_alignment_loss_time           2024-11-02 04:00:05
+  high_ser_error_count              2
```

Comment:
How is high_ser_error different from high_ber_error? Would you give a reference to their defintions?

Reply:
https://github.com/opencomputeproject/SAI/blob/master/inc/saiport.h#L84